feature-contributors/src/main/java/io/github/droidkaigi/confsched2022/feature/contributors/Contributors.kt

```diff
@@ -59,7 +58,7 @@ fun Contributors(
             )
         }
     ) { innerPadding ->
-        Box(modifier = Modifier.padding(innerPadding)) {
+        Box(modifier = modifier) {
```

Comment:
The modifier parameter should not be passed to child composable. But it was originally so. We can make a issue and you can leave it as it is.

Reply:
I believe that it is better to pass on the necessary variables only when they are needed. In this case, `Modifier` is the same situation. So I clean up the modifier as soon as I stopped passing it to grandchildren composable.

Reply:
I think we should always pass modifiers.
We are having the lint for it in https://github.com/DroidKaigi/conference-app-2022/pull/464. So, for now, it is ok. If you want to fix this. You can make a PR!
https://chris.banes.dev/always-provide-a-modifier/